src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/Unichar.cs

```diff
@@ -0,0 +1,76 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System.Collections.Generic;
+using Microsoft.PowerFx.Core.App.ErrorContainers;
+using Microsoft.PowerFx.Core.Errors;
+using Microsoft.PowerFx.Core.Functions;
+using Microsoft.PowerFx.Core.Functions.Delegation;
+using Microsoft.PowerFx.Core.Localization;
+using Microsoft.PowerFx.Core.Types;
+using Microsoft.PowerFx.Core.Utils;
+using Microsoft.PowerFx.Syntax;
+
+namespace Microsoft.PowerFx.Core.Texl.Builtins
+{
+    // Unichar(arg:n) : s
+    // Corresponding Excel function: Unichar
+    internal sealed class UnicharFunction : BuiltinFunction
+    {
+        public override ArgPreprocessor GetArgPreprocessor(int index, int argCount)
+        {
+            return base.GetGenericArgPreprocessor(index);
+        }
+
+        public override bool IsSelfContained => true;
+
+        public override bool IsStateless => true;
+
+        public UnicharFunction()
+            : base("Unichar", TexlStrings.AboutUnichar, FunctionCategories.Text, DType.String, 0, 1, 1, DType.Number)
+        {
+        }
+
+        public override IEnumerable<TexlStrings.StringGetter[]> GetSignatures()
+        {
+            yield return new[] { TexlStrings.UnicharArg1 };
+        }
+    }
+
+    // Unichar(arg:*[n]) : *[s]
+    internal sealed class UnicharTFunction : BuiltinFunction
+    {
+        public override bool IsSelfContained => true;
+
+        public UnicharTFunction()
+            : base("Unichar", TexlStrings.AboutUnicharT, FunctionCategories.Table, DType.EmptyTable, 0, 1, 1, DType.EmptyTable)
+        {
+        }
+
+        public override IEnumerable<TexlStrings.StringGetter[]> GetSignatures()
+        {
+            yield return new[] { TexlStrings.UnicharTArg1 };
+        }
+
+        public override bool CheckTypes(CheckTypesContext context, TexlNode[] args, DType[] argTypes, IErrorContainer errors, out DType returnType, out Dictionary<TexlNode, DType> nodeToCoercedTypeMap)
+        {
+            Contracts.AssertValue(args);
+            Contracts.AssertAllValues(args);
+            Contracts.AssertValue(argTypes);
+            Contracts.Assert(args.Length == argTypes.Length);
+            Contracts.AssertValue(errors);
+            Contracts.Assert(MinArity <= args.Length && args.Length <= MaxArity);
+
+            var fValid = base.CheckTypes(context, args, argTypes, errors, out returnType, out nodeToCoercedTypeMap);
+            Contracts.Assert(returnType.IsTable);
+
+            // Typecheck the input table
+            fValid &= CheckNumericColumnType(context, args[0], argTypes[0], errors, ref nodeToCoercedTypeMap);
+
+            // Synthesize a new return type
+            returnType = DType.CreateTable(new TypedName(DType.String, GetOneColumnTableResultName(context.Features)));
```

Comment:
> GetOneColumnTableResultName(context.Features))) [](http://example.com/codeflow?start=70&length=48)

New functions that return a single-column table all have the schema `*[Value:<type>]` - i.e., the column name is always `Value`, regardless of the features.